Entities/Ingame/ClubFinances.cs

```diff
@@ -74,9 +74,17 @@ public ClubFinances (int memoryAddress, ArraySegment<byte> originalBytes, IVersi
 		}
 
 		public int WeeklyWageBudget {
-			get {
-				return PropertyInvoker.Get<Int32> (ClubFinancesOffsets.WeeklyWageBudget, OriginalBytes, MemoryAddress, DatabaseMode);
-			}
+			get {
+                int value = PropertyInvoker.Get<Int32>(ClubFinancesOffsets.WeeklyWageBudget, OriginalBytes, MemoryAddress, DatabaseMode);
+                int subValue = PropertyInvoker.Get<Int32>(0x4C, OriginalBytes, MemoryAddress, DatabaseMode);
+
+                if (value == 0 || subValue == 0 || subValue < 0 || subValue > 10000)
+                {
+                    return 0;
+                }
+                value = value + subValue;
+                return value;
+            }
 		}
```

Comment:
Can you please explain why you add a value from another offset to the WeeklyWageBudget? Adding up values in the framework will lead to trouble when writing back to the memory is needed (when we implement the setters). It also potentially breaks compatibility with other versions.